editor/plugins/gizmos/gizmo_3d_helper.h

```diff
@@ -42,19 +42,46 @@ class Gizmo3DHelper : public RefCounted {
 	Variant initial_value;
 	Transform3D initial_transform;
 
+private:
+	void cylinder_or_capsule_set_handle(const Vector3 p_segment[2], int p_id, real_t &r_height, real_t &r_radius, Vector3 &r_cylinder_position, bool p_is_capsule);
```

Comment:
```suggestion
	void _cylinder_or_capsule_set_handle(const Vector3 p_segment[2], int p_id, real_t &r_height, real_t &r_radius, Vector3 &r_cylinder_position, bool p_is_capsule);
```
For private methods this is the preferred naming